stockflux-securities-master/src/components/SecuritiesTable.js

```diff
@@ -78,17 +97,57 @@ const SecuritiesTable = ({ location }) => {
                 <div className="securities-table-cell">{item.symbol}</div>
                 <div className="securities-table-cell">{item.name}</div>
                 <div className="securities-table-cell">
-                  <Link to={`/inputform/${item.securityId}`}>
-                    <button className="securities-table-button">
-                      <span className="material-icons">edit</span>
+                  <ToolTip message="Edit">
+                    <Link to={`/inputform/${item.securityId}`}>
+                      <button className="securities-table-button">
+                        <FaPen size={20} />
+                      </button>
+                    </Link>
+                  </ToolTip>
+                  <ToolTip message="Delete">
+                    <button
+                      className="securities-table-button"
+                      onClick={() => deleteSecurity(item.securityId)}
+                    >
+                      <FaTrashAlt size={20} />
+                    </button>
+                  </ToolTip>
+                  <ToolTip message={item.visible ? "Hide" : "Show"}>
+                    <button
+                      className={`securities-table-button ${
+                        item.visible ? "" : "greyed-out"
```

Comment:
Again, try and avoid ternary operations, when one the options is an empty string: "". You can just use && instead.

Reply:
okay changed now